templates/page.ejs

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width">
     <title>Testing Heuristics</title>
     <link href='./style.css' rel='stylesheet'>
-    <link href="https://fonts.googleapis.com/css?family=Taviraj" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Taviraj:400,700" rel="stylesheet">
```

Comment:
It's font weight 300/400 is normal, 700/800 is typically 'bold'
![image](https://cloud.githubusercontent.com/assets/5173131/25841659/e6bdb504-3498-11e7-9ea1-af4bfca9cf95.png)

See https://fonts.google.com/?selection.family=Roboto